flask/packages.py

```diff
@@ -20,6 +21,22 @@
     "downloads": "downloads",
 }
 
+def is_valid_license_identifier(license_str):
+    """
+    Function to check whether the license string is a valid identifier or not.
+
+    Parameters:
+    license_str (str): The SPDX license identifier string to be validated.
+
+    Returns:
+    bool: True if the string is a valid SPDX license identifier, False otherwise.
+    """
+    try:
+        licensing = get_spdx_licensing()
+        licensing.parse(license_str, validate=True)
+        return True
+    except:
+        return False
```

Comment:
Nice, could you add some tests to make sure that the integration keeps working in the future?

Reply:
Yes, I will update the existing tests for upload API because the API has been changed and tests are failing.